presto-docs/pom.xml

```diff
@@ -95,6 +95,16 @@
                             </arguments>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>run-sphinx</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>${project.basedir}/build</executable>
```

Comment:
don't you need to pass (export) `PRESTO_VERSION`?

Reply:
No, that is an override. The Sphinx configuration picks up the version automatically from the POM.